src/text_file_functions.cpp

```diff
@@ -285,7 +283,8 @@ pos_size_type AOApplication::get_element_dimensions(QString p_identifier,
                                                     QString p_file,
                                                     QString p_char)
 {
```

Comment:
You should replace most of this code with a call to `get_design_element`.

Reply:
Resolved as of [`eb988d9`](https://github.com/AttorneyOnline/AO2-Client/pull/302/commits/eb988d920942ed1bcf222d6827f230d3aaa654e7)